client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,48 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakup.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakup = ( event ) => {
```

Comment:
**Nitpick:** There are a few references to "breakup" throughout this PR which brings back painful memories of my teenage years, should we change these to "breakdown" instead?

**Break-up**
<img width="659" alt="Screen Shot 2021-07-28 at 9 44 32 am" src="https://user-images.githubusercontent.com/1527164/127241504-a8ee06b4-9cd3-422a-9dfd-de800f2c399e.png">


**Breakdown**
<img width="634" alt="Screen Shot 2021-07-28 at 9 44 39 am" src="https://user-images.githubusercontent.com/1527164/127241534-07008801-fba4-4b43-9806-f813b4d1e2aa.png">

Reply:
Haha, I like the suggestion, fixed in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/25fc362ff7b03b354fab7b7117e6aeb4ef15ccce 😛